src/sentry/models/auditlogentry.py

```diff
@@ -87,14 +87,10 @@ def save_or_write_to_kafka(self):
         For those silos, this method publishes the attempted audit log write to a durable kafka queue synchronously
         that will eventually be consumed by the control silo.  For the control silo, this method ultimately results
         in a save() call.
-
-        This method is most ideal for shared code paths that may be invoked from either control or region silos,
-        but is not recommended on code paths that should always be invoked from the control silo and depend on the
-        synchronous database access.
         """
-        from sentry.region_to_control.producer import produce_audit_log_entry
+        from sentry.region_to_control.producer import audit_log_entry_service
```

Comment:
Artifact from the PR this is based on (https://github.com/getsentry/sentry/pull/40353/files)